### ISSUE TITLE
Add new option `consistent` to `multiline-ternary`

### ISSUE THREAD

**Issue by evilebottnawi** [eslint/eslint#8602]
**What rule do you want to change?**

multiline-ternary

**Does this change cause the rule to produce more or fewer warnings?**

More

**How will the change be implemented? (New option, new default behavior, etc.)?**

Add option `consistent`. Similar issue https://github.com/eslint/eslint/issues/6488 for `object-curly-newline` rule.

**Please provide some example code that this change will affect:**

```js
// 1 example
test > 1 ? 2 : 3;
// 2 example
test > 1 
    ? 2 
    : 3;
// 3 example
test > 1
    ? 2 : 3;
// 4 example
test > 1 ? 2
    : 3;
```

**What does the rule currently do for this code?**

With `always` value require newlines in 1 example, with `never` require don't using newlines in 2,3 and 4 examples.

**What will the rule do after it's changed?**
If ternary have newline(s) require to using newlines everywhere, if ternary don't have newlines don't report.

**Comment by kaicataldo** [comment]
Is the rationale behind this that you want to enforce consistency in a file but not across a codebase?

**Comment by evilebottnawi** [comment]
@kaicataldo on all my projects :smile: We have small options for `consistently`, but it's really convenient

**Comment by not-an-aardvark** [comment]
Thanks for your interest in improving ESLint. Unfortunately, it looks like this issue didn't get enough support from the team and so I'm closing it. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to reach consensus after a long time tend to never do it, and as such, we close those issues. This doesn't mean the idea isn't interesting, just that it's not something the team can commit to.
